test/client/spec/components/victory-scatter/victory-scatter.spec.js

```diff
@@ -85,6 +102,39 @@ describe("components/victory-scatter", () => {
       const points = wrapper.find(Point);
       expect(points.length).to.equal(30);
     });
+
+    it("renders points in the correct positions", () => {
+      const data = [{x: 0, y: 0}, {x: 2, y: 3}, {x: 5, y: 5}];
+      const wrapper = shallow(
+        <VictoryScatter data={data}/>
+      );
+
+      const viewBoxDimensions = wrapper.prop("viewBox").split(" ");
+      const svgSize = [
+        viewBoxDimensions[2] - viewBoxDimensions[0],
+        viewBoxDimensions[3] - viewBoxDimensions[1]
+      ];
+
+      const points = wrapper.find(Point);
+      const svgCoordinates = points.map(SvgTestHelper.getSvgPointCoordinates);
+
+      // The padding between the edge of the svg and the actual chart area can
+      // be determined by the smallest x or y svg coordinate.
+      const svgPadding = svgCoordinates.reduce((prev, coord) => {
+        return coord[0] < prev ? coord[0] : prev;
+      }, Number.POSITIVE_INFINITY);
+
+      const coordinates = svgCoordinates.map(coord => {
+        return SvgTestHelper.convertSvgCoordinatesToCartesian(
+          coord,
```

Comment:
Can you pull padding out of the props for each point component instead? You might be able to do the same for height and width, though viewBox might complicate that.